python/lsst/jointcal/jointcal.py

```diff
@@ -77,6 +90,7 @@ class JointcalConfig(pexConfig.Config):
         doc = "How to select sources for cross-matching",
         default = "astrometry"
     )
+    # TODO: CmdLineTask has a profile thing built-in: can we tie in to that?
```

Comment:
We have the lsstDebug trick to handle fine-grained turning on of debugging, and something similar could work for profiling.   

In fact, people aren't thrilled about the mechanism that lsstDebug uses, but the replacement for lsstDebug should be able to handle profiling too.